modules/plugins/wfly-10/src/main/resources/META-INF/rhq-plugin.xml

```diff
@@ -8755,245 +8755,413 @@
+      <c:simple-property name="default-security-domain" required="false" type="string" readOnly="false" defaultValue="other" description="The default security domain used by web deployments. The default value is other."/>
+      <c:simple-property name="default-server" required="false" type="string" readOnly="false" defaultValue="default&#45;server" description="The default server to use for deployments. The default value is default&#45;server."/>
+      <c:simple-property name="default-servlet-container" required="false" type="string" readOnly="false" defaultValue="default" description="The default servlet container to use for deployments. The default value is default."/>
+      <c:simple-property name="default-virtual-host" required="false" type="string" readOnly="false" defaultValue="default&#45;host" description="The default virtual host to use for deployments. The default value is default&#45;host."/>
+      <c:simple-property name="instance-id" required="false" type="string" readOnly="false" defaultValue="{EXPRESSION_VALUE=&#36;{jboss.node.name}}"         description="The cluster instance id. The default value is {EXPRESSION_VALUE=&#36;{jboss.node.name}}."/>
+      <c:simple-property name="statistics-enabled" required="false" type="boolean" readOnly="false" defaultValue="false" description="Configures if are statistics enabled. The default value is false."/>
+      <!--
+      There is a configuration node here
+      - Configuration
+        - filter
+          - connection-limit
+          - error-page
+          - gzip
+          - response-header
+          - custom-filter
+          - expression-filter
+          - mod-cluster
+          - response-header
+            - server-header
+              - header-name
+              - header-value
+            - x-powered-by-header
+              - header-name
+              - header-value
+        - handler
+          - file
+            - welcome-content
+              - *more properties*
+          - reverse-proxy
+          
+          ./d2d.sh ... "subsystem=undertow,configuration=filter" connection-limit
+          yields
+           <c:simple-property name="max-concurrent-requests:expr" required="false" type="string" readOnly="false" description="Maximum number of concurrent requests"/>
+           <c:simple-property name="queue-size:expr" required="false" type="string" readOnly="false" description="Number of requests to queue before they start being rejected"/>
+        
+           Should I put a "configuration" service or is more like a group like...
+      <c:group name="child:configuration=filter" displayName="Connection limit">
+        <c:simple-property name="max-concurrent-requests:expr" required="false" type="string" readOnly="false" description="Maximum number of concurrent requests"/>
+        <c:simple-property name="queue-size:expr" required="false" type="string" readOnly="false" description="Number of requests to queue before they start being rejected"/>
```

Comment:
I'm not sure how to put this part, currently looking at the CLI. But I suppose it requires some coding.